bundles/best_practices.rst

```diff
@@ -28,10 +28,11 @@ the guides.
 Bundle Name
 -----------
 
-A bundle is also a PHP namespace. The namespace must follow the `PSR-0`_ or
-`PSR-4`_ interoperability standards for PHP namespaces and class names: it starts
-with a vendor segment, followed by zero or more category segments, and it ends
-with the namespace short name, which must end with a ``Bundle`` suffix.
+A bundle name is also a PHP namespace. The namespace must follow the `PSR-0`_
```

Comment:
I'm against this change, the bundle is a PHP namespace. Not the bundle name perse.